analysis/analysis/analysis.py

```diff
@@ -81,32 +83,58 @@ def parse_args() -> Namespace:
     # ap.add_argument("--", action="store_true", help="")
     ap.add_argument("--devel", action="store_true", help="Devel mode")
     ap.add_argument("--stdout", action="store_true", help="Log to stdout")
-    ap.add_argument("--db-uri", help="Override DB URI", default=DB_URI)
+    ap.add_argument("--db-uri", help="Override DB URI")
+    
     return ap.parse_args()
 
 
-def main() -> None:
+def setup():
     global conf
-    log.info("Analysis starting")
-    # cp = ConfigParser()
-    # with open("/etc/ooni/analysis.conf") as f:
-    #     cp.read_file(f)
 
     conf = parse_args()
+
+    # Set up logs
     if conf.devel or conf.stdout or not has_systemd:
         format = "%(relativeCreated)d %(process)d %(levelname)s %(name)s %(message)s"
         logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, format=format)
-
     else:
         log.addHandler(JournalHandler(SYSLOG_IDENTIFIER="analysis"))
         log.setLevel(logging.DEBUG)
 
     log.info("Logging started")
+
     conf.output_directory = (
         Path("./var/lib/analysis") if conf.devel else Path("/var/lib/analysis")
     )
     os.makedirs(conf.output_directory, exist_ok=True)
 
+    # Parse configs
+    conf_file = DEV_CONF_FILE if conf.devel else CONF_FILE
+
+    # nothing else to do if there's no config to parse
+    if not conf_file.exists():
+        conf.db_uri = DEFAULT_DB_URI
+        return     
+
+    cp = ConfigParser()
+    with conf_file.open("r") as f:
```

Comment:
I would suggest dropping all this config file handling stuff and only allow setting db-uri via the CLI flag `--db-uri`.

Reply:
The reason is that we are potentially adding some extra code paths that can be tricky to debug if something is not working as intended.

Reply:
How do we set up the flag `--db-uri`  when deployed?  I originally used the conf file because it's where we specify configurations in Ansible